src/stores/quiz/getters.ts

```diff
@@ -0,0 +1,5 @@
+import { quizAnswer } from './state'
+
+const quizGetters = {
```

Comment:
A getter should always return something, and usually is a computed. Try to replicate the behaviour in other getters in admin project